src/Language/PureScript/Docs/Types.hs

```diff
@@ -184,6 +184,12 @@ data DeclarationInfo
 
 instance NFData DeclarationInfo
 
+extractKindSig :: DeclarationInfo -> Maybe (Text, Type')
+extractKindSig (DataDeclaration dtype ks _) = fmap (P.showDataDeclType dtype,) ks
+extractKindSig (TypeSynonymDeclaration ks _ _) = fmap ("type",) ks
+extractKindSig (TypeClassDeclaration ks _ _ _) = fmap ("class",) ks
+extractKindSig _ = Nothing
```

Comment:
Another temporary solution. Ideally, we would be using `keywordType` or something like that, so that any changes (if ever made) only need to made in one location.